app/styles/_variables.scss

```diff
@@ -0,0 +1,875 @@
+$bootstrap-sass-asset-helper: false !default;
```

Comment:
Instead of having an 875-line file with comment blocks describing different areas, just have multiple files.
`app/styles/variables/colors.scss`
`app/styles/variables/scaffolding.scss`
`app/styles/variables/sconography.scss`
etc.